rpc/errors.go

```diff
@@ -95,3 +100,15 @@ func (e *shutdownError) Code() int {
 func (e *shutdownError) Error() string {
 	return "server is shutting down"
 }
+
+// connectionClosedError is thrown when the connection is closed
+type connectionClosedError struct {
+}
+
+func (e *connectionClosedError) Code() int {
```

Comment:
It implements the `RPCError` interface in the same package. IMHO it is not necessary to document that it implements this interface.